pallets/pips/src/lib.rs

```diff
@@ -173,13 +171,11 @@ impl_checked_inc!(PipId);
 impl PipId {
     /// Converts a PIP ID into a name of a PIP scheduled for execution.
     pub fn execution_name(&self) -> Vec<u8> {
-        use polymesh_common_utilities::constants::schedule_name_prefix::*;
```

Comment:
any particular reason for this to be here?

Reply:
Nope.  The import doesn't need to be here.